lib/aruba/generators/script_file.rb

```diff
@@ -0,0 +1,42 @@
+# Aruba
+module Aruba
+  # ScriptFile
+  class ScriptFile
+    def initialize(opts = {})
+      @path        = opts[:path]
+      @content     = opts[:content]
+      @interpreter = opts[:interpreter]
+    end
+
+    def call
+      script_content = script_starts_with_shebang? ? @content : "#{header}#{@content}"
```

Comment:
:-) No need for this test. Please remove line 12 and change the code like this.

~~~
Aruba.platform.write_file(@path, "#{header}#{@content}")
~~~